src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -538,10 +537,6 @@ const formConfig = {
                 customTitle: ' ', // prevent <dl> around the schemaform-field-container
                 confirmRemove: true,
               },
-              'ui:errorMessages': {
-                minItems: 'Must have at least one applicant listed.',
-                maxItems: `You can add up to ${MAX_APPLICANTS} in a single application. If you need to add more than ${MAX_APPLICANTS} applicants, you need to submit a separate application for them.`,
-              },
```

Comment:
The min item error message doesn't display because the form defaults to having a single blank applicant, and the `maxItems` message is overridden by the default warning so it never displayed either. Decided to just pull these out while I was in here.